pages/index.js

```diff
@@ -18,7 +18,7 @@ function Home({ categories }) {
     <div className="d-flex flex-column main-container">
       <Header categories={categories} logs={data && data.logs ? data.logs : []} />
       <HomeCarousel />
-      <main className="container my-4 flex-grow-1">
+      <div className="container my-4 flex-grow-1">
```

Comment:
what's the difference between className and class attributes? Is there one?

Reply:
`className` is React specific, it becomes `class`